sds_data_manager/stacks/create_schema_stack.py

```diff
@@ -0,0 +1,66 @@
+"""SdpDatabase Stack"""
+# Installed
+import pathlib
+
+import aws_cdk
+from aws_cdk import CustomResource, Environment, Stack
+from aws_cdk import (
+    aws_ec2 as ec2,
+)
+from aws_cdk import (
+    aws_lambda as lambda_,
+)
+from aws_cdk import aws_lambda_python_alpha as lambda_alpha_
+from aws_cdk import (
+    aws_secretsmanager as secrets,
+)
+from aws_cdk import custom_resources as cr
+from constructs import Construct
+
+
+class CreateSchema(Stack):
+    """Stack for creating schema creation lambda"""
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        env: Environment,
+        db_secret_name: str,
+        vpc: ec2.Vpc,
+        vpc_subnets,
+        rds_security_group,
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, construct_id, env=env, **kwargs)
+
+        schema_create_lambda = lambda_alpha_.PythonFunction(
+            self,
+            id="CreateMetadataSchema",
+            function_name="create-schema-hoyt-test-2",
+            entry=str(
+                pathlib.Path(__file__).parent.joinpath("..", "lambda_code").resolve()
+            ),
+            index="SDSCode/create_schema.py",
+            handler="lambda_handler",
+            runtime=lambda_.Runtime.PYTHON_3_9,
+            timeout=aws_cdk.Duration.minutes(15),
```

Comment:
I think Greg's been commenting about reducing this timeout duration and it's a good idea. Can you change it to somewhat close to how long it takes to complete task in this lambda?